src/layout/Tabs/TabsTheme.ts

```diff
@@ -2,43 +2,71 @@ export interface TabsTheme {
   base: string;
   list: {
     base: string;
+    indicator: string;
+    divider: string;
+    variant: {
+      primary: {
+        indicator: string;
+        divider: string;
+      };
+      secondary: {
+        indicator: string;
+        divider: string;
+      };
+    };
     tab: {
       base: string;
+      button: string;
       selected: string;
       disabled: string;
+      variant: {
+        primary: {
+          button: string;
+        };
+        secondary: {
+          button: string;
+        };
+      };
     };
-    indicator: string;
   };
   panel: string;
 }
 
 const baseTheme: TabsTheme = {
   base: 'flex flex-col',
   list: {
-    base: 'flex text-center flex-wrap -mb-px position-relative',
-    tab: {
-      base: 'rounded-none border-b-2 border-transparent',
-      selected: '',
-      disabled: 'cursor-not-allowed opacity-50'
+    base: 'flex text-center flex-wrap -mb-px',
+    indicator: 'bg-primary absolute bottom-0 left-0 right-0',
+    divider: 'w-full h-px border-0',
+    variant: {
+      primary: {
+        indicator: 'h-[4px]',
```

Comment:
i feel like i go back and forth on whether these types of values should be fixed or if it should be using `h-1` so that if the base font size is changed, the design would scale too.

Reply:
oh true I think we should go rem